default/scripting/buildings/CULTURE_ARCHIVES.focs.py

```diff
@@ -0,0 +1,63 @@
+from buildings.buildings import SPECIES_LIKES_OR_DISLIKES_BUILDING_STABILITY_EFFECTS
+from focs._effects import (
+    Capital,
+    Described,
+    EffectsGroup,
+    Happiness,
+    NamedReal,
+    Object,
+    OwnedBy,
+    Planet,
+    Population,
+    SetEmpireMeter,
+    SetTargetIndustry,
+    SetTargetResearch,
+    Source,
+    Target,
+    Unowned,
+    Value,
+)
+from macros.base_prod import BUILDING_COST_MULTIPLIER, INDUSTRY_PER_POP
+from macros.priorities import TARGET_AFTER_2ND_SCALING_PRIORITY
+
+try:
+    from focs._buildings import *
+except ModuleNotFoundError:
+    pass
+
+BuildingType(  # type: ignore[reportUnboundVariable]
+    name="BLD_CULTURE_ARCHIVES",
+    description="BLD_CULTURE_ARCHIVES_DESC",
+    buildcost=200 * BUILDING_COST_MULTIPLIER,
+    buildtime=1,
+    tags="ANTIQUATED",
+    location=Described(description="NOWHERE", condition=~All),
```

Comment:
It is `IsAnyObject`.

Reply:
Or `None` since it's `~`?

Reply:
It was `location = Described description = "NOWHERE" condition = Not All` before not sure why.